### PR TITLE
Contributing section in README.md

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Auf CLA in github readme hinweisen](https://trello.com/c/YXR6Zufe/603-auf-cla-in-github-readme-hinweisen)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Tests
+
+Our CI runs several tests regarding code formatting, static lints, and also our test suite of unit and integration tests.
+Please provide tests that accompany your code.
+Our coverage tool will automatically tell you how much of our contribution is covered by tests.
+
+### Your PR is merged!
+
+Congratulations! The Geo Engine team thanks you for your contribution. ✨
```
Comment (michaelmattig) [inline CONTRIBUTING.md:50]: maybe 
```suggestion
Congratulations! The Geo Engine community thanks you for your contribution. ✨
```

**Review by michaelmattig — CHANGES_REQUESTED** [review]
.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md]: d82a873

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+### CLA
+
+We require you to sign a contributor license agreement before we can accept your code contribution.
+You can visit [www.geoengine.de/cla](https://www.geoengine.de/cla/) to get more information.
+
+### Code reviews
+
+The core team usually makes a code review on each pull request.
+We address all issues and try to give helpful feedback.
+You can either rebuttal a comment or provide a fix.
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md]: d82a873

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+All PRs run through some steps.
+
+### CLA
+
+We require you to sign a contributor license agreement before we can accept your code contribution.
+You can visit [www.geoengine.de/cla](https://www.geoengine.de/cla/) to get more information.
+
+### Code reviews
+
+The core team usually makes a code review on each pull request.
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md]: d82a873

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+If you file a bug report, please tell us what the expected outcome would be and how to reproduce the bug.
+It would be nice if you create, for instance, a failing test with the expected outcome.
+
+### 📣 Discussion
+
+If you want to discuss new features before implementing them, feel free to start a discussion and get feedback on your idea.
+
+### ❓ Support
+
+In case you have difficulties using Geo Engine, feel free to ask for help!
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md:22]: d82a873

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+There are three main types of issues.
+
+### 🐞 Bug report
+
+If you file a bug report, please tell us what the expected outcome would be and how to reproduce the bug.
+It would be nice if you create, for instance, a failing test with the expected outcome.
+
+### 📣 Discussion
+
+If you want to discuss new features before implementing them, feel free to start a discussion and get feedback on your idea.
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md]: d82a873

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (michaelmattig) [reply CONTRIBUTING.md:22]: I would leave support@ for paying(?) customer/users. How about contribute@ or dev@ or maybe just simply info@ ?

[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md:22]: f129362

[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply CONTRIBUTING.md]: d7c276c

**Pull request merged**